codecov.yml

```diff
@@ -1,5 +1,5 @@
 codecov:
-  require_ci_to_pass: yes
+  require_ci_to_pass: true
```

Comment:
this wasn't even a valid option in the config yaml

Reply:
lol good catch, since when is yes/no valid for a boolean param...

Reply:
my IDE told me :)

Reply:
and TG for IDEs

Reply:
python has a (read: had a) now deprecated built-in library function (`strtobool`) for doing just that - converting from yes/no (and other string values) to `True` and `False` - 😬